### DISCUSSION TITLE
facing issue in kibana after upgrading SO to 2.3.160

### DISCUSSION THREAD

**Discussion by G-onejalal** [Security-Onion-Solutions/securityonion#8809] [Unsupported Versions]
Hi All, 

we are facing one issue with kibana after upgarding SO to 2.3.160 version. whenever i am clicking on any of the kibana dashboard field its throwing me this error. 
![image](https://user-images.githubusercontent.com/102476291/192106291-ab076ecb-0e6a-4ba7-9c57-0dffd57d0639.png)

anyone have any idea about this issue.

**Comment by dougburks** [comment]
What version did you upgrade from?

If you were upgrading to Elastic 8, did you follow all instructions at https://docs.securityonion.net/en/2.3/soup.html#elastic-8?
